packages/wallet-sdk/src/relay/MobileRelay.ts

```diff
@@ -0,0 +1,41 @@
+import { isInIFrame } from "../util";
+import { WalletLinkRelay, WalletLinkRelayOptions } from "./WalletLinkRelay";
+import { CancelablePromise } from "./WalletSDKRelayAbstract";
+import { RequestEthereumAccountsResponse } from "./Web3Response";
+
+export class MobileRelay extends WalletLinkRelay {
+  private _useMobileWalletLink: boolean;
```

Comment:
Instead of typescript privates we can start using javascript private methods

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

```ts
  #useMobileWalletLink: boolean;
```

Reply:
thanks for flagging that. 
will address that during overall refactor.